source/wifi/wifi_hal.c

```diff
@@ -22,6 +22,11 @@
 * Licensed under the BSD-3 license
 */
 
+/*
```

Comment:
Hi Tomasz: Please change from "This component includes" to "This file includes...".  If all the Plume code is all in one place, these two lines should go in front of that code instead at the start of the file.
Thanks,
Martin Hughes, RDK CMF Compliance Team